peer.h

```diff
@@ -52,6 +57,7 @@ typedef struct dtls_peer_t {
 
   dtls_peer_type role;       /**< denotes if this host is DTLS_CLIENT or DTLS_SERVER */
   dtls_state_t state;        /**< DTLS engine state */
+  int16_t optionalHandshakeMessage; /**< optional next handshake message, DTLS_HT_NO_OPTIONAL_MESSAGE, if no optional message is expected. */
```

Comment:
Please follow C coding conventions. An identifier thus would merely look like this: `optional_handshake_message`.

Reply:
Too much java ...